### ISSUE TITLE
ComputationError (ZeroDivisionError) on split-descriptive-statistics

### ISSUE THREAD

**Issue by severo** [huggingface/datasets-server#2631]
For  mozilla-foundation/common_voice_6_1 / dv / other, we currently have error `ComputationError` due to `ZeroDivisionError`. cc @polinaeterna

**Comment by polinaeterna** [comment]
it seems to be a parquet file with 0 rows

**Comment by polinaeterna** [comment]
...so it divides by zero lol. not sure what kind of response should be there. this seems to be an extremely rare case

**Comment by polinaeterna** [comment]
> I think we should handle the case

Agree, should we do changes in other steps too? Like it doesn't make sense to create parquet files or duckdb index files in such cases I think

**Comment by severo** [comment]
Yes, you're right, maybe we want to handle the "no rows" case in a global PR
